packages/jest-each/src/bind.js

```diff
@@ -21,7 +21,10 @@ export default (cb: Function) => (...args: any) => (
   test: Function,
 ): void => {
   if (args.length === 1) {
-    const table: Table = args[0];
+    const table: Table = args[0].every(Array.isArray)
+      ? args[0]
+      : args[0].map(entry => [entry]);
```

Comment:
should we be clever and only wrap those not already arrays? or would that never happen in practice?

Reply:
I think it would look strange:

```js
test.each([
  ['foo'],
  'bar',
  ['baz'],
  'qux'
])('do something %s', word => {});
```

But in saying that it would be pretty trivial to just wrap everything that isn't an array in an array. I'm happy to update if you think it makes more sense :smile:

Reply:
Imagine something like this:
```js
test.each([
  ['foo', 'bar'],
  'foobar',
  ['baz', 'zzz', 'woke', 'blazingmeansgood'],
  'qux'
])('do something %s', words => {});
```

Reply:
Again, not sure if it's useful at all or if we should just throw

Reply:
Yeah, throwing on mismatching `length`s sounds good